test/FunctionalTests/Microsoft.OData.Core.Tests/ODataMessageWriterTests.cs

```diff
@@ -845,6 +845,31 @@ public async Task WriteMetadataDocumentAsync_WorksForJsonCsdl_WithNoSynchronousI
 }", payload);
         }
 
+        [Fact]
+        public async Task WriteLargeMetadataDocumentAsync_WorksForJsonCsdl_WithNoSynchronousIOSupport()
+        {
+            // Arrange
+            IEdmModel largeModel = GetLargeEdmModel();

```

Comment:
If the largeModel doesn't change, we don't need to create the whole model for each test.
You can make it as static field and use it for all test cases.

Reply:
Done